src/components/i18n/MenuIntlProvider.tsx

```diff
@@ -5,7 +5,7 @@ import { messages_es } from "./es";
 import messages_en from "./en";
 import { flattenMessages } from "./utils";
 
-const defaultLanguage = "es";
+export const defaultLanguage = "es";
```

Comment:
I would revert this change.

Reply:
you say it for this: `Adding a default configuration file` [like this](https://github.com/FromDoppler/doppler-editors-webapp/blob/main/src/default-configuration.ts)

Reply:
I think that default configuration is not related with this line.

`defaultLanguage` should only be used inside `MenuIntlProvider.tsx`, so, you do not need to export it.